frontend/src/constants.js

```diff
@@ -1,11 +1,19 @@
 export const TAGS = ["Climate", "Social Justice", "Education", "LGBTQ"];
 
-export const BILL_STATUSES = {
-    INTRODUCED: 'Introduced',
-    IN_COMMITTEE: 'In Committee',
-    ON_FLOOR_CALENDAR: 'On Floor Calendar',
-    PASSED_SENATE: 'Passed Senate',
-    PASSED_ASSEMBLY: 'Passed Assembly',
-    DELIVER_TO_GOVERNOR: 'Deliver to Governor',
-    SIGNED_BY_GOVERNOR: 'Signed By Governor',
-};
+export const BILL_STATUSES = [
+  'Introduced',
+  'In Senate Committee',
+  'In Assembly Committee',
+  'Senate Floor Calendar',
+  'Assembly Floor Calendar',
+  'Passed Senate',
+  'Passed Assembly',
+  'Delivered to Governor',
+  'Signed By Governor',
+  'Vetoed',
+];
+
+export const SEARCH_QUERY_KEY_MAP =  {
+  STATUS: 'status.statusdesc',
```

Comment:
```suggestion
  STATUS: 'status.statusDesc',
```

Not working for me otherwise.